domains/nhavy.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "github": "VzTong"
```

also, add a contact method
```suggestion
    "username": "VzTong"
```